### PR TITLE
feat: vue calendar

### PR DESCRIPTION
**Pull request by dotaitch**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description

* Implement a Vue wrapper.
* Add guides for a Vue wrapper.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by adhrinae — COMMENTED** [review]
고생하셨어요. `index.d.ts` 쪽은 모듈 이름 확인 부탁드려요.

File: apps/vue-calendar/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import type Vue from 'vue';
+import type Calendar from '@toast-ui/calendar';
+
+declare module '@toast-ui/calendar' {
```
[the remark this answers is not in the document]
Comment (dotaitch) [reply apps/vue-calendar/index.d.ts:4]: 아이코 넵 맞아요 수정해두겠습니다

File: apps/vue-calendar/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  getRootElement(): HTMLDivElement;
+  getInstance(): Calendar; 
+}
+
+declare namespace tui {
+  export class VueCalendar extends Vue {
+    getRootElement(): HTMLDivElement;
+    getInstance(): Calendar; 
+  }
+}
```
Comment (lja1018) [inline apps/vue-calendar/index.d.ts:14]: ts라 그런지 newline 적용이 안됬네요

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다.

**Comment by Tragio** [comment]
Really happy to see this fantastic work. 😃 It would be nice to mention the Vue/Nuxt supported versions as the ecosystem is currently divided between versions 2 and 3. Since this plugin does not rely on any specific Vue features there shouldn't be a problem, but to avoid confusion would be nice to have that information.

**Comment by adhrinae** [comment]
@Tragio You're right. we don't have a plan to support Vue 3 at this moment since all other products in the organization doesn't use Vue 3 yet.

**Pull request merged**